drivers/network/dd/nvnet/netnv.inf

```diff
@@ -0,0 +1,288 @@
+HKR, Ndi\params\ChecksumOffload,       ParamDesc, 0, %CO%
+HKR, Ndi\params\ChecksumOffload,       type,      0, "enum"
+HKR, Ndi\params\ChecksumOffload,       default,   0, "1"
+HKR, Ndi\params\ChecksumOffload\enum,  "0",       0, "%No%"
+HKR, Ndi\params\ChecksumOffload\enum,  "1",       0, "%Yes%"
+
+HKR, Ndi\params\LargeSendOffload,       ParamDesc, 0, %LS%
+HKR, Ndi\params\LargeSendOffload,       type,      0, "enum"
+HKR, Ndi\params\LargeSendOffload,       default,   0, "1"
+HKR, Ndi\params\LargeSendOffload\enum,  "0",       0, "%No%"
+HKR, Ndi\params\LargeSendOffload\enum,  "1",       0, "%Yes%"
+
+[NVNET_Jumbo]
+HKR, Ndi\params\JumboSize,             ParamDesc, 0, %JF%
+HKR, Ndi\params\JumboSize,             type,      0, "int"
+HKR, Ndi\params\JumboSize,             default,   0, "1514"
+HKR, Ndi\params\JumboSize,             min,       0, "1514"
+HKR, Ndi\params\JumboSize,             max,       0, "9014"
+HKR, Ndi\params\JumboSize,             step,      0, "1"
+HKR, Ndi\params\JumboSize,             base,      0, "10"
+
+[NVNET_VLAN]
+HKR, Ndi\params\Priority,              ParamDesc, 0, %PP%
+HKR, Ndi\params\Priority,              type,      0, "enum"
+HKR, Ndi\params\Priority,              default,   0, "1"
+HKR, Ndi\params\Priority\enum,         "0",       0, "%No%"
+HKR, Ndi\params\Priority\enum,         "1",       0, "%Yes%"
+
+HKR, Ndi\params\VlanTag,               ParamDesc, 0, %VT%
+HKR, Ndi\params\VlanTag,               type,      0, "enum"
+HKR, Ndi\params\VlanTag,               default,   0, "1"
+HKR, Ndi\params\VlanTag\enum,          "0",       0, "%No%"
+HKR, Ndi\params\VlanTag\enum,          "1",       0, "%Yes%"
+
+HKR, Ndi\params\VlanID,                ParamDesc, 0, %VI%
+HKR, Ndi\params\VlanID,                type,      0, "long"
+HKR, Ndi\params\VlanID,                default,   0, "0"
+HKR, Ndi\params\VlanID,                min,       0, "0"
+HKR, Ndi\params\VlanID,                max,       0, "4095"
+HKR, Ndi\params\VlanID,                step,      0, "1"
+HKR, Ndi\params\VlanID,                base,      0, "10"
+
+[NVNET_Service_Inst]
+ServiceType   = 1
+StartType     = 3
+ErrorControl  = 1
+ServiceBinary = %12%\nvnet.sys
+LoadOrderGroup = NDIS
+
+[NVNET_EventLog]
+AddReg = NVNET_EventLog_AddReg
+
+[NVNET_EventLog_AddReg]
+HKR, , EventMessageFile, 0x00020000, "%%SystemRoot%%\System32\netevent.dll"
+HKR, , TypesSupported,   0x00010001, 7
+
+;-------------------------------- STRINGS -------------------------------
+
+[Strings]
+ReactOS = "ReactOS Team"
+
+nVidiaMfg = "nVidia"
+
+OM = "Optimization Mode"
+M1 = "Dynamic"
+M2 = "CPU"
+M3 = "Throughput"
+NA = "Network Address"
+SD = "Speed & Duplex"
+Auto = "Auto"
+CO = "Checksum Offload"
+LS = "Large Send Offload"
+FC = "Flow Control"
+FRx = "Rx Pause"
+FTx = "Tx Pause"
+FBoth = "Rx & Tx Pause"
+No = "No"
+Yes = "Yes"
+PP = "Packet Priority"
+VT = "VLAN Tagging"
+VI = "VLAN ID"
+JF = "Jumbo Frame"
+
+NVNET.DeviceDesc = "nVidia nForce Ethernet Controller"
```

Comment:
Maybe it's worth to add other supported adapter descriptions here like nForce2, nForce3, etc. (referenced in `nic.c` line 699) But I can do this myself later, not critical :)

Reply:
>other supported adapter descriptions

It seems that the word 'nForce' always shows up in the device manager page if I install the vendor's NIC driver